staging_service/utils.py

```diff
@@ -58,7 +58,8 @@ def validate_path(username: str, path: str=''):
         full_path = os.path.join(Path._DATA_DIR, user_path)
         metadata_path = os.path.join(Path._META_DIR, user_path)
         name = os.path.basename(path)
-        jgi_metadata = os.path.join(os.path.dirname(full_path), '.' + name + '.jgi')
+        jbi_uuid = name.split('.')[0]
```

Comment:
Is this the format they are using?  So if the file name was 123.456.fastq, the metadata file is .123.metadata?  That could be a problem.

Reply:
@scanon As current procedure, all JGI imported file will have a 16bit_jgi_uuid append to the file. Such as 16bit_jgi_uuid.123.456.fastq. Then the metadata is the 16bit_jgi_uuid.metadata. Erik is thinking about a new procedure includes changing the file name. But before it's implemented, the file name always has the 16bit_jgi_uuid.

Reply:
Got it.

Reply:
Hasn't JGI already removed the JAMO record id prefix from files and metadata? It should now just be filename and .filename.jgi, no?

Reply:
I am afraid not. I just tried to stage a JGI file. It's still having that JAMO id suffix.

I went to 'public' -> 'JGI Public Data (TEST)' and choose a random file and hit 'Stage'. It appears in my staging area with the JAMO id suffix.

Reply:
Ah, you should be testing current jgi->staging behavior with the jgi search tool.
The old jgi search and staging tool in the Narrative is now quite out of date and works against the old jgi gateway (surprised it still works, actually!) When no filename is supplied, the api must still use the old file naming style. In the jgi search tool, we pass either the original or the user-supplied filename, which overrides the old behavior. 

I'll contact Harika @ JGI about removing the old behavior altogether -- I don't think it is something we want to support at all at this point.